### PR TITLE
updated keyvault jca changelog for local jre keystore certificates

### PR DESCRIPTION
**Pull request by michaelqi793**

### CODE REVIEW

File: sdk/keyvault/azure-security-keyvault-jca/CHANGELOG.md [hunk 1]
```diff
@@ -1,7 +1,8 @@
 # Release History
 
 ## 1.0.0-beta.8 (Unreleased)
-
+### New Features
+- Support the local JRE key store certificates. [#21845](https://github.com/Azure/azure-sdk-for-java/pull/21845)
```
Comment (chenrujun) [inline sdk/keyvault/azure-security-keyvault-jca/CHANGELOG.md:5]:
```
Load JRE key store certificates to AzureKeyVault key store. ([#21845](https://github.com/Azure/azure-sdk-for-java/pull/21845))
```

File: sdk/spring/azure-spring-boot-starter-keyvault-certificates/CHANGELOG.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
 # Release History
 
 ## 3.0.0-beta.8 (Unreleased)
-
+- Support the local JRE key store certificates. [#21845](https://github.com/Azure/azure-sdk-for-java/pull/21845)
```
Comment (chenrujun) [inline sdk/spring/azure-spring-boot-starter-keyvault-certificates/CHANGELOG.md:4]: Same here.

**Review by chenrujun — CHANGES_REQUESTED** [review]

File: sdk/keyvault/azure-security-keyvault-jca/CHANGELOG.md (hunk 1 above)
Comment (michaelqi793) [reply sdk/keyvault/azure-security-keyvault-jca/CHANGELOG.md]: Done

File: sdk/spring/azure-spring-boot-starter-keyvault-certificates/CHANGELOG.md [hunk 3]
```diff
@@ -1,7 +1,7 @@
 # Release History
 
 ## 3.0.0-beta.8 (Unreleased)
-
+- Load JRE key store certificates to AzureKeyVault key store. ([#21845](https://github.com/Azure/azure-sdk-for-java/pull/21845))
```
Comment (chenrujun) [inline sdk/spring/azure-spring-boot-starter-keyvault-certificates/CHANGELOG.md:4]: We need `### New Features` here.

**Review by chenrujun — CHANGES_REQUESTED** [review]

File: sdk/spring/azure-spring-boot-starter-keyvault-certificates/CHANGELOG.md (hunk 3 above)
Comment (michaelqi793) [reply sdk/spring/azure-spring-boot-starter-keyvault-certificates/CHANGELOG.md:5]: Done

**Review by chenrujun — APPROVED** [review]
LGTM.

**Pull request merged**